modules/@apostrophecms/doc-type/ui/apos/components/AposDocContextMenu.vue

```diff
@@ -389,7 +389,18 @@ export default {
         } else {
           this.$emit('close', doc);
         }
+      } else if (doc.slug.startsWith('/')) {
+        // If doc is a page and `this.current` is `null` (ie. we are on Pages manager modal),
+        // load the whole page to duplicate the page AND its whole content:
+        const page = await apos.http.get(`/api/v1/@apostrophecms/page/${doc._id}`, {
+          busy: true
+        });
+        console.log('LOAD FULL DOC');
+        console.log('page', page);
+        Object.assign(doc, page);
```

Comment:
Use `Object.assign` in order not to lose references

Reply:
Seems strange that we only handle pages. Pieces should have the same issue here or am I missing something?

Reply:
unlike pages, pieces have all the info:

<img width="1675" alt="image" src="https://github.com/apostrophecms/apostrophe/assets/8301962/56debcc0-f6c9-4186-be50-efb339f4d483">